package.json

```diff
@@ -10,6 +10,6 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/CommunityGD32Cores/ArduinoCore-GD32"
+    "url": "https://github.com/arcayi/ArduinoCore-GD32"
```

Comment:
For the merge, this should be reverted to the original one.

Reply:
```suggestion
    "url": "https://github.com/CommunityGD32Cores/ArduinoCore-GD32"
```